### PR TITLE
Added Basic Authentication

### PR DESCRIPTION
**Pull request by SanketDhabarde**
### Added basic authentication with Mockbee
- Add dummy user login
- Add auth context to store user
### Login 
Screenshot (185)

### Signup
Screenshot (187)

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -17,7 +18,9 @@ ReactDOM.render(
       <ProductFilterProvider>
```
Comment (jeetbhalani15) [inline src/index.js:18]: suggestion! u can use one provider which  contains all other providers

**Review by jeetbhalani15 — APPROVED** [review]
Readable code &
nice ui too.

**Review by Naveenchandar — APPROVED** [review]
Clean and nice folder structure.
Good usage of components and react hooks.

File: src/index.js (hunk 1 above)
Comment (SanketDhabarde) [reply src/index.js:18]: Wow. That's a nice idea. never thought about it. Thanks man

**Pull request merged**